proc-macros/Cargo.toml

```diff
@@ -15,7 +15,7 @@ proc-macro = true
 [dependencies]
 proc-macro2 = "1.0"
 quote = "1.0"
-syn = { version = "1.0", default-features = false, features = ["extra-traits", "full", "visit", "parsing"] }
+syn = { version = "1.0", default-features = false, features = ["extra-traits", "full", "visit", "parsing", "printing", "clone-impls", "proc-macro"] }
```

Comment:
Interesting!

Reply:
These features leaked from the workspace but not sure why it's was catched now